packages/flutter_tools/test/daemon_test.dart

```diff
@@ -168,7 +155,7 @@ void main() {
       commands.close();
     });
 
-    _testUsingContext('daemon.callServiceExtension', () async {
+    testUsingContext('daemon.callServiceExtension', () async {
```

Comment:
"ext.flutter.debugPaint via service extension without an appId should report an error"